packages/fx-core/resource/package.nls.json

```diff
@@ -1,79 +1,79 @@
+  "core.provision.confirmEnvAndCostNotice": "Costs may incur according to the usage. Do you want to provision resources for the '%s' environment using the accounts listed above?",
+  "core.provision.confirmEnvOnlyNotice": "Do you want to provision resources for the '%s' environment using the account listed above?",
+  "core.deploy.confirmEnvNotice": "Do you want to deploy resources for the '%s' environment?\nAzure account (%s) with subscription (%s) will be used for deployment.",
+  "core.deploy.confirmEnvNoticeV3": "Do you want to deploy resources for the '%s' environment?",
   "core.provision.successAzure": "Azure resources successfully provisioned in the cloud.",
   "core.provision.successNotice": "'%s' successfully provisioned in the cloud.",
-  "core.provision.failNotice": "fail to provision '%s' in the cloud!",
-  "core.provision.StartNotice": "[%s] Start provisioning. It could take several minutes.",
+  "core.provision.failNotice": "failed to provision '%s' in the cloud.",
+  "core.provision.StartNotice": "[%s] Provisioning started. It could take several minutes.",
   "core.provision.configurationFinishNotice": "[%s] Configuration finished.",
-  "core.provision.ProvisionFinishNotice": "[%s] Provision finished!",
+  "core.provision.ProvisionFinishNotice": "[%s] Provisioning finished.",
   "core.provision.viewResources": "View provisioned resources",
-  "core.provision.subscription.NotFoundParam": "The subscription '%s' for '%s' environment is not found in the current account, please use the right Azure account or check the subscription parameter.",
-  "core.provision.subscription.NotFoundConfig": "The subscription '%s' for '%s' environment is not found in the current account, please use the right Azure account or check the '%s' file.",
-  "core.provision.subscription.NotFoundState": "The subscription '%s' for '%s' environment is not found in the current account, please use the right Azure account.",
-  "core.provision.subscription.NotFound": "The subscription '%s' is not found in the current account, please use the right Azure account.",
-  "core.provision.subscription.failToSelect": "Failed to select subscription",
+  "core.provision.subscription.NotFoundParam": "The subscription '%s' for the '%s' environment was not found in the current account, please use the correct Azure account or check the subscription parameter.",
+  "core.provision.subscription.NotFoundConfig": "The subscription '%s' for the '%s' environment was not found in the current account, please use the correct Azure account or check the '%s' file.",
+  "core.provision.subscription.NotFoundState": "The subscription '%s' for the '%s' environment was not found in the current account, please use the correct Azure account.",
+  "core.provision.subscription.NotFound": "The subscription '%s' was not found in the current account, please use the correct Azure account.",
+  "core.provision.subscription.failToSelect": "Failed to select the subscription",
```

Comment:
thiking this line also follow the pattern that the actual name of the subscription should be displayed? missing '%s'?

Reply:
That might require a code change wherever this string is being used and is outside the scope of these audits.